test/test_regression_cli.py

```diff
@@ -109,7 +116,11 @@ def test_csv_regression(dataset, expected, tmpdir):
 
 @pytest.mark.parametrize("dataset, expected", download_md5)
 def test_download_regression(dataset, expected):
-    """Check for regression for a particular dataset downloaded only"""
+    """Check for regression for a particular dataset downloaded only
+
+    move the dataset script downloaded to the .retriever file before download
```

Comment:
Thanks Ethan, I intend to leave them the way they are for this PR/commit. However we are going to change then for #318